packages/terra-dropdown-button/src/_Dropdown.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FocusTrap from 'focus-trap-react';
+import Hookshot from 'terra-hookshot';
+import DropdownList from './_DropdownList';
+
+const propTypes = {
+  /**
+   * Callback to tell the parent it should close the dropdown
+   */
+  requestClose: PropTypes.func.isRequired,
+  /**
+   * Ref to attach the dropdown to
+   */
+  targetRef: PropTypes.func.isRequired,
+  /**
+   * Whether or not the dropdown should be open
+   */
+  isOpen: PropTypes.bool.isRequired,
+  /**
+   * The Options that should be displayed in the dropdown
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * @private
+   * Width to set the dropdown to. Used when `isBlock` is true. Unset means to autosize.
+   */
+  width: PropTypes.string,
+};
+
+const Dropdown = ({
+  requestClose, isOpen, targetRef, children, width,
+}) => (
+  <Hookshot
+    isOpen={isOpen}
+    isEnabled
+    targetRef={targetRef}
+    attachmentBehavior="flip"
```

Comment:
nit: could we possibly make use of the enum [here](https://github.com/cerner/terra-framework/blob/master/packages/terra-hookshot/src/Hookshot.jsx#L19) for this?

Reply:
Because those objects are arrays the best I could do is `{Hookshot.attachmentBehaviors[1]}` which is certainly worse than `"flip"`

Reply:
Oh no, my bad.